enos/modules/docker_openssh_server_ca_key/main.tf

```diff
@@ -105,7 +106,7 @@ resource "enos_local_exec" "wait" {
     docker_container.openssh_server
   ]
 
-  inline = ["timeout 30s bash -c 'until ssh -t -t -i ${var.private_key_file_path} -p 2222 -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no -o IdentitiesOnly=yes ${var.target_user}@localhost hostname; do sleep 2; done'"]
+  inline = ["timeout 60s bash -c 'until ssh -t -t -i ${var.private_key_file_path} -p 2222 -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no -o IdentitiesOnly=yes ${var.target_user}@localhost hostname; do sleep 2; done'"]
```

Comment:
Also mode this modification as the startup for this openssh container can take a little longer. Longer-term, I want to find a better solution for this, but this is the quickest way for now.